Rubberduck.CodeAnalysis/Inspections/Abstract/IsMissingInspectionBase.cs

```diff
@@ -0,0 +1,61 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using Antlr4.Runtime;
+using NLog;
+using Rubberduck.Inspections.Abstract;
+using Rubberduck.Parsing;
+using Rubberduck.Parsing.Grammar;
+using Rubberduck.Parsing.Symbols;
+using Rubberduck.Parsing.VBA;
+
+namespace Rubberduck.Inspections.Inspections.Abstract
+{
+    public abstract class IsMissingInspectionBase : InspectionBase
+    {
+        private readonly ILogger _logger = LogManager.GetCurrentClassLogger();
```

Comment:
can be `static`

Reply:
In an abstract base class?

Reply:
Unless I'm mistaken, `GetCurrentClassLogger` would return a logger for `IsMissingInspectionBase` anyway. But yeah best avoid `static` in a base class.